samples/client.c

```diff
@@ -816,32 +831,17 @@ usage(char *argv0)
             basename(argv0));
 }
 
-/*
- * Normally each time the source client (QEMU) would read migration data from
- * the device it would send them to the destination client. However, since in
- * our sample both the source and the destination client are the same process,
- * we simply accumulate the migration data of each iteration and apply it to
- * the destination server at the end.
- *
- * Performs as many migration loops as @nr_iters or until the device has no
- * more migration data (pending_bytes is zero), which ever comes first. The
- * result of each migration iteration is stored in @migr_iter.  @migr_iter must
- * be at least @nr_iters.
- *
- * @returns the number of iterations performed
- */
```

Comment:
Is this comment not relevant any more?

Reply:
I'm not sure why I removed it, it still applies. I'll put it back.